tasks_management/services.py

```diff
@@ -181,6 +183,9 @@ def _adjust_create_task_data(self, obj_data):
                 obj_data[key] = str(obj_data[key])
         return obj_data
 
+    def _data_for_json_ext_create(self, obj_data):
```

Comment:
Do we need to pass 'obj_data' argument in method?

Reply:
removed

Reply:
can't remove because it is being called with the argument in task_create

Reply:
ok